src/basics/task2.js

```diff
@@ -0,0 +1,38 @@
+import { pipeline } from 'node:stream'
+import fs from 'node:fs';
+import csvToJSON from 'csvtojson';
+import path from 'node:path';
+const fsPromises = fs.promises;
+
+
+const outputFolderPath = path.join('./output');
+const outputFilePath = path.join(outputFolderPath, 'convertedCSV.txt');
+const csvFilePath = path.resolve('src/assets/csv', 'CSVExample.csv');
+const csvDelimiter = ';';
+
+
+async function createFolder(path) {
+    try {
```

Comment:
no need to use the `try/catch` in such a way. As usual, the helper-functions should not know how to handle the errors, it should be achieved on the very top level of the application, i.e. in main/entry point of the app.